### PR TITLE
Add initial copy of Emergency Shutdown strategy

### PR DESCRIPTION
**Pull request by claudijd**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -145,3 +146,32 @@ The role of the incident commander for Wormhole includes the following minimum o
...
+
+The rationale for this decision included the following considerations:
+
+- Introduces new trust assumptions for the protocol specific to shutdown events.
+- Adds additional engineering overhead to design, develop, and maintain shutdown capability for each supported run-time.
+- Adds attack surface to Wormhole related smart contracts, which could introduce new bugs.
+- Adds gas cost on every transaction for a feature that will be used maybe once a year.
+- Makes assumptions about where in the code existential crisis may surface.
+
+As an alternative approach, Wormhole has evolved it's emergency shutdown strategy to leverage the existing upgrade authority via governance to temporarily patch smart contracts for vulnerabilities when/if they occur (eg. temporarily disabling the affected function) and a long-term bug fix is not readily available.
```
Comment (kcsongor) [inline SECURITY.md:162]:
```suggestion
As an alternative approach, Wormhole has evolved its emergency shutdown strategy to leverage the existing upgrade authority via governance to temporarily patch smart contracts for vulnerabilities when/if they occur (eg. temporarily disabling the affected function) and a long-term bug fix is not readily available.
```

File: SECURITY.md [hunk 2]
```diff
@@ -145,3 +146,32 @@ The role of the incident commander for Wormhole includes the following minimum o
 - Establish a private incident document, where the problem, timeline, actions, artifacts, lessons learned, etc. can be tracked and shared with responders.
 - When an incident is over, host a [retrospective](https://en.wikipedia.org/wiki/Retrospective) with key responders to understand how things could be handled better in the future (this is a no blame session, the goal is objectively about improving Wormhole's readiness and response capability in the future).
 - Create issues in relevant ticket trackers for actions based on lessons learned.
+
+## Emergency Shutdown
+
+The Wormhole project has evaluated the concept of having an on-chain safety feature, which could allow Wormhole smart contracts to be paused during a state of existential crisis.  However, after careful consideration the project has chosen to not make an on-chain safety feature that would allow an emergency shutdown capability at this time.
```
[the remark this answers is not in the document]
Comment (claudijd) [reply SECURITY.md:152]: agreed, will adjust.

**Review by bruce-riley — DISMISSED** [review]

**Review by njkumr — APPROVED** [review]

File: SECURITY.md [hunk 3]
```diff
@@ -145,3 +146,32 @@ The role of the incident commander for Wormhole includes the following minimum o
...
+- Maintain the same trust assumptions for existential events
+- Allow selective shutdown of only the affected code (while leaving everything else fully functional)
+- No additional attack surface (only less attack surface during existential crisis)
+- No additional gas cost paid by users of Wormhole
+- Makes no assumptions about where the bug is, giving full utility to surgically remove it where it is
+- No additional process or keys to distribute or manage for Guardians
+
+The caveats of such an approach include the following:
+
+- Speed to shutdown is limited by speed to develop the temporary bug fix
```
[the remark this answers is not in the document]
Comment (kcsongor) [reply SECURITY.md]: I think the shutdown should be prepared for all components, along the lines of https://github.com/wormhole-foundation/wormhole/pull/1938

**Review by tbjump — APPROVED** [review]
lgtm well written

**Pull request merged**